### PR TITLE
Added Wireguard noop support

### LINKED ISSUE #502: Frame WG service as Node PT with Noops

### PR DESCRIPTION
**Pull request by soffokl**
Closes #502

### CODE REVIEW

File: cmd/commands/service/command.go [hunk 1]
```diff
@@ -28,6 +28,7 @@ import (
 	service_noop "github.com/mysteriumnetwork/node/services/noop"
 	service_openvpn "github.com/mysteriumnetwork/node/services/openvpn"
 	openvpn_service "github.com/mysteriumnetwork/node/services/openvpn/service"
+	service_wireguard "github.com/mysteriumnetwork/node/services/wireguard"
```
Comment (tadovas) [inline cmd/commands/service/command.go]: There is clearly something wrong with package structure inside openvpn :( service_openvpn and openvpn_service? I know it's out of this PR scope, but please don't do it with wireguard too :)

File: services/wireguard/service.go [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+	// out, err := exec.Command("ip", "address", "add", "dev", "wg0", "192.168.100.1/24").CombinedOutput()
+
+	// TODO configure wireguard interface.
+	// client, err := wireguardctrl.New()
+	// if err != nil {
+	// 	return Config{}, err
+	// }
+
+	// TODO wireguard device configuration like private key, listen port, peer list should survive through restarts.
+	// TODO we need to have some persistent storage for it.
```
Comment (zolia) [inline services/wireguard/service.go]: We can create separate ticket for that. Its not high priority IMHO.

File: services/wireguard/service.go [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+	// client.ConfigureDevice("wg0", wgtypes.Config{
+	// 	PrivateKey: "8C6Pp0cI2tgLeYOMVbnMMLl/zN2blFK+OWGaIxX0bHY=",
+	// 	ListenPort: 52820,
+	// 	Peers:      nil,
+	// })
+
+	// TODO if the wireguard interface already configured we can get required parameters from it.
+	// device, err := client.Device("wg0")
```
Comment (zolia) [inline services/wireguard/service.go]: WG device name should be selected only after listing all active devices. Then choosing free number. We should know which device constitutes which session.

**Comment by tadovas** [comment]
After sync with master :)

**Pull request merged**